packages/xsnap/src/xsnap.js

```diff
@@ -73,24 +79,32 @@ export function xsnap(options) {
   const vatExit = defer();
 
   const args = snapshot ? ['-r', snapshot] : [];
+  if (meteringLimit !== undefined) {
```

Comment:
for consistency, consider:

```js
const args = [...(snapshot ? ['-r', snapshot] : []),
              ...(meteringLimit ? ['-l', `${meteringLimit}`] : [])]
```

I assume setting `meteringLimit` to 0 is not a useful case.

Reply:
That's more functional, but I have been criticised in the past for using rest arguments with expressions.

A `meteringLimit` of 0 is exactly how one needs to disable metering, so I've updated the conditional accordingly.